### DISCUSSION TITLE
I cannot understand that RECEIPIENT

### DISCUSSION THREAD

**Discussion by NerdHusky** [FiloSottile/age#505] [UX reports]
I wanted to try a public key first:
```
age -o id_ed25519.pub.age -r "ssh-ed25519 pub"  id_ed25519.pub
age: error: malformed SSH recipient: "ssh-ed25519 pub": ssh: no key found
age: report unexpected or unhelpful errors at https://filippo.io/age/report
```
Maybe I did wrong.
Now let's try the "real deal". Sure this can´t go wrong....
```
age -o id_ed25519.age -r "ssh-ed25519"  id_ed25519
age: error: malformed SSH recipient: "ssh-ed25519": ssh: no key found
age: report unexpected or unhelpful errors at https://filippo.io/age/report
```

And so I came here, hopeless.

**Comment by str4d** [comment]
The `-r` option takes the actual text of the recipient pubkey, because for a native age recipient that is a string `age1....`. It is not a type selector option like `ssh-keygen -t ed25519`. So if you want to use it this way, you'd need to take the entire content line from your `id_ed25519.pub` file and pass it as a string option for `-r`.

Instead, what you should do is use the `-R` option, which takes a path to a file containing a list of recipients. That is exactly what `id_ed25519.pub` is, so your command should be:

```
age -o file-to-encrypt.age -R  id_ed25519.pub file-to-encrypt
```

**Reply by NerdHusky** [reply]
Thank you very much man for taking the time for answering me!! This is SO helpful, thank you!!
